src/psyclone/tests/psyir/backend/sympy_writer_test.py

```diff
@@ -317,11 +319,14 @@ def test_sympy_writer_get_symbol_and_map(expr, sym_map, fortran_reader):
     # Ignore the converted expressions here, they are tested elsewhere
     _ = writer([expr])
     assert writer._sympy_type_map.keys() == sym_map.keys()
+    # Repeat when the supplied Node does not have a parent tree
```

Comment:
What difference could that make? Are e.g. the symbols now not defined anymore??

Reply:
It caused a crash where I tried to lookup the symbol (L343 of sympy_writer.py).